Ukoly/DU_6/Piskvorky.py

```diff
@@ -0,0 +1,74 @@
+from random import randrange
+
+WINNER = None
+
+
+def evaluation(string):
+    for _ in string:
+        if 'xxx' in string and 'ooo' not in string:
+            WINNER = '"x"'
+        elif 'ooo' in string and 'xxx' not in string:
+            WINNER = '"o"'
+        elif '-' in string and ('xxx' not in string or 'ooo' not in string):
+            WINNER = None
+        elif '-' not in string and \
+            ('xxx' not in string or 'ooo' not in string):
+            WINNER = 'no one'
+    if WINNER is not None:
+        print('The winner is ' + WINNER)
+    return(WINNER)
+
+
+def move_of_player(field, sign):
+    number_of_field = input('Pick your position. ')
+    while True:
+        number_of_field = str(int(number_of_field) - 1)
+        if not number_of_field.isnumeric():
+            print('Use only numbers in range. ')
+            number_of_field = input('Pick your position. ')
+        elif int(number_of_field) >= 20 or int(number_of_field) < 0:
+            print('The number is too big or too small. ')
+            number_of_field = input('Pick your position. ')
```

Comment:
Opäť veríme užívateľovi, že nám sem nezadá nezmysel... :smile_cat: 

Fun fact: Užívateľ by to tam určite zadal aj keby mu to tam napíšeš veľkým a červeným...